src/coreclr/jit/emitarm64.cpp

```diff
@@ -2010,6 +2018,24 @@ const char* emitter::emitPredicateRegName(regNumber reg)
     return pRegNames[index];
 }
 
+//------------------------------------------------------------------------
+// emitPredicateAsCounterRegName: Returns a predicate as counter register name.
+//
+// Arguments:
+//    reg - A predicate register.
+//
+// Return value:
+//    A string that represents a predicate as counter register name.
+//
+const char* emitter::emitPredicateAsCounterRegName(regNumber reg)
```

Comment:
I was thinking if we should just pass `pType` in existing `emitPredicateRegName()` method instead of having separate method for `pnRegNames`.

Reply:
That's much simpler...... And done.